index.html

```diff
@@ -14,10 +14,14 @@
                 <input id="search-jobs-input" placeholder="Search jobs" type="search" />
             </form>
             <section>
+                <p>just a little change</p>
```

Comment:
Can you please add span around this text?
Also I made a suggestion, you can apply my suggestion by clicking on "commit suggestion"
After you click on commit, it will actually make a new commit, so you will need to pull the newest changes by running
git pull origin feature/search-2
after that make the request changes (add a span) and then push those changes again to origin feature/search-2
come back here to see them
I will then merge your PR and then you can do this:
git checkout development
git pull upstream development 
that's it :)
BTW you opened the PR against the master branch and it's supposed to be against development, so please click edit and change the base branch
```suggestion
                <p>Just a little change</p>
```